src/components/tree.vue

```diff
@@ -207,7 +207,10 @@ export default {
         throw new ReferenceError('newNode is required but undefined')
       }
       if (typeof newNode === 'string') {
-        addnode = {title: newNode}
+        addnode = {
+          id: (Math.random().toString(36)+'00000000000000000').slice(2, 7), // Generate random id of 5 characters
```

Comment:
if u use a random id, then no initialization ID how to deal with?

Reply:
if the ID may exist may not exist, then the existence of id here is what is the meaning?

Reply:
I tend to the user's own incoming ID， and you